xarray/core/dataset.py

```diff
@@ -9204,9 +9209,11 @@ def pad(
             (stat_length,) or int is a shortcut for before = after = statistic
             length for all axes.
             Default is ``None``, to use the entire axis.
-        constant_values : scalar, tuple or mapping of hashable to tuple, default: 0
-            Used in 'constant'.  The values to set the padded values for each
-            axis.
+        constant_values : scalar, tuple, mapping of hashable to tuple or
+            mapping of hashable to mapping of hashable to tuple, default: 0
```

Comment:
we might need a backslash to format this correctly:
```suggestion
        constant_values : scalar, tuple, mapping of hashable to tuple or \
            mapping of hashable to mapping of hashable to tuple, default: 0
```

Reply:
Thanks! Will add this too.